block/manager_test.go

```diff
@@ -80,7 +80,8 @@ func TestInitialState(t *testing.T) {
 			assert := assert.New(t)
 			logger := log.TestingLogger()
 			dalc := getMockDALC(logger)
-			agg, err := NewManager(key, conf, c.genesis, c.store, nil, nil, dalc, nil, logger)
+			dumbChan := make(chan struct{})
```

Comment:
Why is this a dumb Channel? :D

Reply:
because nobody is listening to it, it's just for testing.